### PR TITLE
First draft

### PR DESCRIPTION
**Pull request by joshpapermaster**

### CODE REVIEW

File: _summaries/clevert2015fast_1.md [hunk 1]
```diff
@@ -1,19 +1,49 @@
...
+- The improvement over LReLU and PReLU is that ELU allows the gradient to smoothly approach zero as the input becomes more negative. 
+- This is important because it maintains a consistent method for reducing noise. 
+- The 2 previous functions linearly introduce negative values, while ELU brings them in exponentially. This allows for a deactivation state of inputs similar to ReLU, while maintaining negative values. 
+
+
+* include better looking math to represent ELU
+* include image from paper of differences of activation functions
+
+
+ELU performs well on the major image recognition sets they tested on. The authors proved ELU had significantly lower activation means and training loss than other activation functions when testing with the MNIST dataset. At the time of their evaluations and publication, ELU held the best test score of CIFAR-100 and second best of CIFAR-10. The CIFAR-100 test error used in their evaluation is 24.28%, which is soundly lower than the previous best of 27.62%.
```
Comment (DartingMelody) [inline _summaries/clevert2015fast_1.md:40]: @joshpapermaster one more suggestion. .....functions when testing with the MNIST dataset.) Thus, comparatively converging faster in training. In testing, due to the non-linear activation for negative input, it's comparatively slower than ReLU and variants.

**Review by DartingMelody — COMMENTED** [review]
@joshpapermaster I have put some comments, can you please review them.

**Comment by joshpapermaster** [comment]
> @joshpapermaster Can you add - because of a reduced bias shift effect.

Hey, @DartingMelody sorry it's not showing me which line you were referring to here

**Comment by DartingMelody** [comment]
Hello @joshpapermaster, can you check this link and see you are able to view the comments for the line. https://github.com/joshpapermaster/review-assignments/commit/b2f753e1074d732d85596a5f49d3be416addece6#commitcomment-55870110 . Also, clicking on _summaries/clevert... (after DartingMelody commented on, in any comment headline) should redirect to this link.

**Comment by joshpapermaster** [comment]
@DartingMelody Thanks for your review, I've updated my PR. Please let me know if you think I'm missing anything.

File: _summaries/clevert2015fast_1.md [hunk 2]
```diff
@@ -25,25 +28,27 @@ ELU maintains the nice properties of ReLU by maintaining f(x) = x for x >= 0.
...
+ELU performs well on the major image recognition sets they tested on. The authors proved ELU had significantly lower activation means and training loss than other activation functions when testing with the MNIST dataset. Thus, ELU comparatively converges faster during training. However, due to the non-linear activation, it was comparitively slower than ReLU and variants during testing. At the time of their evaluations and publication, ELU held the best test score of CIFAR-100 and second best of CIFAR-10. The CIFAR-100 test error used in their evaluation is 24.28%, which is soundly lower than the previous best of 27.62%.
 
 
-Their evaluations proved ELU is adaptable to many different image recognition CNN's. ELU on its own scored higher than ReLU, LReLU, PReLU, and SReLU with and without batch normalization.
+Their evaluations proved ELU is adaptable to many different image recognition CNN's. ELU without batch normalization performed comparatively better than ReLU with and without batch normalization. Batch normalization did not imporve the performance of ELU in their testing.
```
Comment (DartingMelody) [inline _summaries/clevert2015fast_1.md:48]: @joshpapermaster thanks for the changes. One last doubt I have does ELU performs better than ReLU without batch normalization because in the paper it is explicitly mentioned that "we show that ELU networks perform better than ReLU networks with batch normalization". Also, there is a typo(imporve) in the last sentence. Thanks again.

Comment (joshpapermaster) [reply _summaries/clevert2015fast_1.md]: @DartingMelody Thank you! I believe that using batch normalization improved the performance of ReLU in their evaluations, but to be safe I have removed that part of it.

**Review by DartingMelody — APPROVED** [review]
